### ISSUE TITLE
Improve layout on mobile

### ISSUE THREAD

**Issue by RobJN** [osmbe/osm-welcome-belgium#34]
I'm finding it too small on mobile. Hopefully the layout can be improved for mobile users.

Also it looks like the table has an extra column but I couldn't get the table to scroll over to the right to show this.

![Screenshot_20210830-171235](https://user-images.githubusercontent.com/5060210/131370551-c4240cfd-7a1a-49ac-aff2-3b1593fd06b5.png)

**Comment by jbelien** [comment]
You're absolutely right, I didn't do anything yet for mobile display but I'm using Tailwind CSS so it's definitely not an issue to improve that.
It's now added to my to-do list! 👍
